<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CBrain</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Start</string>
    </property>
    <widget class="QMenu" name="menuEinstellungen_2">
     <property name="title">
      <string>Einstellungen</string>
     </property>
     <addaction name="actionNetzwerkordner_aendern"/>
     <addaction name="actionBenutzer_verwalten"/>
     <addaction name="actionProgrammeigene_Datenbank"/>
    </widget>
    <addaction name="menuEinstellungen_2"/>
    <addaction name="actionBenutzer_wechsen"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionTestfunktion"/>
   </widget>
   <widget class="QMenu" name="menuModule">
    <property name="title">
     <string>Module</string>
    </property>
    <addaction name="actionKeinModul"/>
    <addaction name="separator"/>
    <addaction name="actionModulArtikel"/>
    <addaction name="actionModulLager"/>
    <addaction name="actionModulLieferanten"/>
    <addaction name="actionModulMaterialliste"/>
    <addaction name="actionModulPersonal"/>
    <addaction name="actionModulProjekte"/>
   </widget>
   <widget class="QMenu" name="menuModule_Admin">
    <property name="title">
     <string>Module Admin</string>
    </property>
    <addaction name="actionTabelleneditor"/>
    <addaction name="actionModulBackup"/>
   </widget>
   <addaction name="menuEinstellungen"/>
   <addaction name="menuExtras"/>
   <addaction name="menuModule"/>
   <addaction name="menuModule_Admin"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNetzwerkordner_aendern">
   <property name="text">
    <string>Wurzelverzeichnis Ã¤ndern</string>
   </property>
  </action>
  <action name="actionBenutzer_wechsen">
   <property name="text">
    <string>Benutzer wechsen</string>
   </property>
  </action>
  <action name="actionBenutzer_verwalten">
   <property name="text">
    <string>Benutzer verwalten</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionProgrammeigene_Datenbank">
   <property name="text">
    <string>Programmeigene Datenbank</string>
   </property>
  </action>
  <action name="actionTestfunktion">
   <property name="text">
    <string>Testfunktion</string>
   </property>
  </action>
  <action name="actionTabelleneditor">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Tabelleneditor</string>
   </property>
  </action>
  <action name="actionKeinModul">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>kein</string>
   </property>
  </action>
  <action name="actionModulArtikel">
   <property name="text">
    <string>Artikel</string>
   </property>
  </action>
  <action name="actionModulLieferanten">
   <property name="text">
    <string>Lieferanten</string>
   </property>
  </action>
  <action name="actionModulLager">
   <property name="text">
    <string>Lager</string>
   </property>
  </action>
  <action name="actionModulBackup">
   <property name="text">
    <string>Backup</string>
   </property>
  </action>
  <action name="actionModulProjekte">
   <property name="text">
    <string>Projekte</string>
   </property>
  </action>
  <action name="actionModulPersonal">
   <property name="text">
    <string>Personal</string>
   </property>
  </action>
  <action name="actionModulMaterialliste">
   <property name="text">
    <string>Materialliste</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
